### DISCUSSION TITLE
Rasberry pi rasbian lite(32bit) support? alternative?

### DISCUSSION THREAD

**Discussion by wieluk** [coder/code-server#2541] [Q&A]
Hello I am trying to install code-server on my raspberry pi there is running a lot of stuff so i tried on a clean image with docker and portainer. But the whole system feels so slow when doing it like this. I also tried on ubuntu server it worked a lot better then in docker. Is there a way to install on rasbian 32bit ? I didn't try yarn or npm yet is it worth trying ? Or will there ever be a  armhf release anytime? 

Thanks for your help

**Comment by code-asher** [comment]
Installing with yarn should work but I haven't tried it myself.

We're tracking a release for 32-bit arm here:
https://github.com/cdr/code-server/issues/1337
